gaek/ndb_json.py

```diff
@@ -131,6 +154,10 @@ def default(self, obj):
     if obj_type not in NDB_TYPE_ENCODING and hasattr(obj, '__metaclass__'):
       obj_type = obj.__metaclass__
     fn = NDB_TYPE_ENCODING.get(obj_type)
+
+    if not self.fetch_models and ALTERNATIVE_NDB_TYPE_ENCODING.has_key(obj_type):
```

Comment:
Not sure about addressing this one in this PR. We can add it after.

I'm not sure how users will specify the function, do you mean by name or constant?